src/test/scala/mesosphere/marathon/integration/AppDeployIntegrationTest.scala

```diff
@@ -396,6 +396,29 @@ class AppDeployIntegrationTest
     pingTask(taskUpdate2).entityString should be(s"Pong $appId\n")
   }
 
+  test("stop (focefully delete) a deployment") {
+    Given("a new app that is not healthy")
+    val appId = testBasePath / "failing"
+    val app = appProxy(appId, "v1", instances = 1, withHealth = false).
```

Comment:
I copied and pasted code that wasn't very nice apparently. I just updated the commit and fixed the test I copied from =).